src/main/scala/org/broadinstitute/hail/check/Gen.scala

```diff
@@ -34,8 +43,8 @@ object Gen {
     a
   }
 
-  def getPartition(parts: Int) : Gen[Array[Int]] = Gen { p => partition(p.rng, p.size, parts) }
-  def getSize : Gen[Int] = Gen { p => p.size }
+  def partition(parts: Int) : Gen[Array[Int]] = Gen { p => partition(p.rng, p.size, parts) }
+  def size : Gen[Int] = Gen { p => p.size }
```

Comment:
`Gen.sized(f: (Int) => Gen[T]): Gen[T]` was the intended way to get the size, although I like this since it works with for comprehensions and "keeps it in the Monad".  It looks like `Gen.sized` and `Gen.size.flatMap` are the same.